processor/src/main/java/org/mapstruct/ap/internal/model/MappingMethod.java

```diff
@@ -58,6 +58,8 @@
      *
      * @param method method
      * @param existingVariableNames existingVariableNames
+     * @param beforeMappingReferences
+     * @param afterMappingReferences
      */
     protected MappingMethod(Method method, Collection<String> existingVariableNames,
```

Comment:
I suppose I'd rather remove the entire comment. Doesn't provide any information really.